### PR TITLE
added more social icons

### PR DESCRIPTION
**Pull request by Dishebh**
## Description
Added more social icons as per the updated [Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=3712%3A1878&viewport=45%2C155%2C1.0786082744598389)

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Dishebh** [comment]
@dhruvkb @panchovm please see if it solves the issue?

**Review by dhruvkb — CHANGES_REQUESTED** [review]
This is awesome! Just a few followup questions:
- Did you use IcoMoon to generate these?
  - If yes, did you refer to [the Changing Icons guide](https://deploy-preview-88--cc-fonts.netlify.app/?path=/docs/fonts-contribution--page#changing-icons)?
    - If yes, were you able to follow the guide?
    - If not, please compare your steps to those in the guide.
  - If yes, please update the `VocabularyIcons.json` file to enable future contributors to build on top of your changes.
  - If yes, please ensure you added the ligatures for all the new icons.
  - If not, what tool did you use?
    - Does that tool offer a better experience?
- Did you ensure that the brand colors are on point?
  - Did you use https://brandcolors.net/?
    - If not, what site did you use? 
      - Does that site offer a better experience?

**Comment by Dishebh** [comment]
@dhruvkb Yes, I went through the [contribution guidelines](https://deploy-preview-88--cc-fonts.netlify.app/?path=/docs/fonts-contribution--page#changing-icons) and used the [IcoMoon App](https://icomoon.io/) to add icons. The guide is absolutely beginner-friendly and so I could get along with it. But, in my view, there was less mention of `VocabularyIcons.json` and how you should / could proceed to update it. This is someplace that, if worked upon, could become more insightful for the beginners. 
I, also, didn't use [Brand Colors](https://brandcolors.net/) website initially but some random another website. But I see that this website has much more to offer and has more descriptive and vivid work. And so I updated the brand colors from this website.

**Comment by dhruvkb** [comment]
@Dishebh please open a new issue to increase the prominence of `VocabularyIcons.json` in the documentation.

File: src/styles/vocabulary_icons.scss [hunk 1]
```diff
@@ -101,13 +107,25 @@ $linkedin-brand: #0077b5;
 $facebook-brand: #3b5998;
 $pinterest-brand: #bd081c;
 $slack-brand: #7c2852;
+$reddit-brand: #ff4500;
+$github-brand: #333333;
+$whatsapp-brand: #25d366;
+$tumblr-brand: #35465c;
+$sms-brand: #4aae20;
```
Comment (dhruvkb) [inline src/styles/vocabulary_icons.scss:114]: Which color did you choose here?

**Review by dhruvkb — APPROVED** [review]
Just one thing is a little unclear.

File: src/styles/vocabulary_icons.scss (hunk 1 above)
Comment (Dishebh) [reply src/styles/vocabulary_icons.scss:114]: Ahh! Chose a color similar to whatsapp here 😅, since I was not sure what to use.

**Comment by Dishebh** [comment]
> 
> 
> @Dishebh please open a new issue to increase the prominence of `VocabularyIcons.json` in the documentation.

Sure, will do! 😄 I am also planning to apply for GSoD this summer with CC. Can you please let me know (the platform) where should I share the draft proposal with the org?  Thanks!

**Comment by dhruvkb** [comment]
@Dishebh you can reach out to the mentors and interact with other aspirants on [our community Slack](https://opensource.creativecommons.org/community/#slack).

**Pull request merged**